### PR TITLE
Complete HW-10 using Spring

### PR DESCRIPTION
**Pull request by kolosokvit**

### CODE REVIEW

File: src/main/java/ticketservice/dao/TicketDao.java [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
+import org.springframework.stereotype.Component;
+import ticketservice.ticket.Ticket;
+import ticketservice.ticket.TicketType;
 
+import javax.sql.DataSource;
 import java.sql.*;
 import java.util.ArrayList;
 import java.util.List;
 
+@Component
```
Comment (AtticSiol) [inline src/main/java/ticketservice/dao/TicketDao.java]: for DAO level it is better to use specific annotation: `@Repository`

File: src/main/java/ticketservice/dao/UserDao.java [hunk 2]
```diff
@@ -1,37 +1,38 @@
...
 
-import jdbc.CustomDataSource;
-import user.User;
+import org.springframework.stereotype.Component;
+import ticketservice.user.User;
 
+import javax.sql.DataSource;
 import java.sql.*;
 
+@Component
```
Comment (AtticSiol) [inline src/main/java/ticketservice/dao/UserDao.java]: for DAO level it is better to use specific annotation: `@Repository`

File: src/main/java/ticketservice/dao/UserDao.java [hunk 3]
```diff
@@ -1,37 +1,38 @@
-package dao;
+package ticketservice.dao;
 
-import jdbc.CustomDataSource;
-import user.User;
+import org.springframework.stereotype.Component;
+import ticketservice.user.User;
 
+import javax.sql.DataSource;
 import java.sql.*;
```
Comment (AtticSiol) [inline src/main/java/ticketservice/dao/UserDao.java]: avoid squashed imports

File: src/main/java/ticketservice/dao/TicketDao.java (hunk 1 above)
Comment (kolosokvit) [reply src/main/java/ticketservice/dao/TicketDao.java]: Fixed

File: src/main/java/ticketservice/dao/UserDao.java (hunk 2 above)
Comment (kolosokvit) [reply src/main/java/ticketservice/dao/UserDao.java]: Fixed

File: src/main/java/ticketservice/dao/UserDao.java (hunk 3 above)
Comment (kolosokvit) [reply src/main/java/ticketservice/dao/UserDao.java]: Fixed

**Review by AtticSiol — APPROVED** [review]

**Pull request merged**